### DISCUSSION TITLE
Ability to have clickable links on related entries in 'field' context.

### DISCUSSION THREAD

**Discussion by GaryReckard** [craftcms/cms#9657] [Ideas] [answered]
I have a situation where it would be pretty convenient to be able to, while viewing the edit page for a user or entry, click an entry in an Entries or Users relationship field to get to its own edit page.

These related entries are clickable on an Index page, but not when displayed in a field.
This area of `CP::elementHtml()` only makes it a link in the 'index' context. 
https://github.com/craftcms/cms/blob/4c99bae64caf7d9b1a93c612a38206b07091e3f0/src/helpers/Cp.php#L391-L400
... and I think that makes a lot of sense as a default. Accidentally clicking out of an edit page when you might have unsaved changes could be a pain. So perhaps when in 'field' context, they could open a new tab, or pop up a confirm dialog if the page has unsaved changes.

Some custom JS got me what I need, but just wanted to share this idea.

**Comment by brandonkelly** [comment, marked as answer]
As of Craft 3.7, it’s now possible to access related element’s primary edit pages, by double-clicking on them to access their editor slideout, and then clicking on this icon in the header:

![Screenshot of an editor slideout, with a circle around the “external” button in the header](https://user-images.githubusercontent.com/47792/127662125-a8b4a0bb-45fe-4bb2-b3d2-98d25decbac3.png)
